### ISSUE TITLE
It looks like there is no way to see my expert profile / status?

### ISSUE THREAD

**Issue by NullVoxPopuli** [airpair/airpair-com-v1-issue-tracker#68]
from the dashboard, there doesn't seem to be a way to see even if I am signed up as an expert. :-\

**Comment by jkresner** [comment]
Correct. Hold tight, you'll see expert stuff reappearing toward the end of the month.

The old pages are still working, though not super helpful:

https://www.airpair.com/be-an-expert

I'll give you a shout when the new stuff starts appearing.
